packages/pds/src/auth.ts

```diff
@@ -105,13 +122,17 @@ export class ServerAuth {
     return header.slice(BEARER.length)
   }
 
-  verifyToken(token: string, scope?: AuthScopes, options?: jwt.VerifyOptions) {
+  verifyToken(
+    token: string,
+    scopes: AuthScope[],
+    options?: jwt.VerifyOptions,
+  ): jwt.JwtPayload {
     try {
       const payload = jwt.verify(token, this._secret, options)
       if (typeof payload === 'string' || 'signature' in payload) {
         throw new InvalidRequestError('Malformed token', 'InvalidToken')
       }
-      if (scope && payload.scope !== scope) {
+      if (scopes.length > 0 && !scopes.includes(payload.scope)) {
```

Comment:
I don't think it matters in any code we have kicking around, but I'm tempted to treat empty `scopes` as always invalid rather than always valid.